subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r36/BuildProgressCrossVersionSpec.groovy

```diff
@@ -73,4 +76,37 @@ class BuildProgressCrossVersionSpec extends ToolingApiSpecification {
         events.operation('Task :runInProcess').descendant('My in-process worker action')
         events.operation('Task :runForked').descendant('My forked worker action')
     }
+
+    def "when running a build then root build progress event is 'Run build'"() {
```

Comment:
I removed the superfluous TAPI tests. `BuildOperationExecutorIntegrationTest` now covers build operation events at configuration time.

It can't assert for the `Run build` build operation as it is emitted around `buildStarted`/`buildFinished` hooks.